frontend/src/components/ExpenseItem.js

```diff
@@ -0,0 +1,69 @@
+import React, {Component, PropTypes} from 'react';
+import moment from 'moment';
+import EXPENSE_ICONS from '../constants/expense_icons';
+import Currency from './Currency';
+
+export default class ExpenseItem extends Component {
+  static propTypes = {
+    expense: PropTypes.object.isRequired,
+    user: PropTypes.object
+  };
+
+  static defaultProps = {
+    expense: {},
+    user: {
+      name: 'Anonymous',
+      avatar: '/static/images/default_avatar.svg'
+    }
+  };
+
+  _getStatus(expense) {
+    if (!!expense.reimbursedAt) {
+      return 'Reimbursed';
+    }
+
+    if (expense.approvedAt && expense.approved) {
+      return 'Approved';
+    }
+
+    if (expense.approvedAt && !expense.approved) {
+      return 'Rejected';
+    }
+
+    return 'Pending';
```

Comment:
@asood123 We should make a virtual attribute in the backend for that in the future